docs/user-documentation/adding-object-to-collection-in-deux.md

```diff
@@ -0,0 +1,19 @@
+Adding an object to a collection in Deux
+========================================
```

Comment:
Ok ... i was using it as a search/replace term, but will swap in Islandora 7.x-2.x